docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -87,12 +119,15 @@ on GPUs.
 
 --8<-- "docs/website/docs/guides/deployment-configurations/snippets/_iree-import-onnx-mobilenet.md"
 
-Then run the following command to compile with the `rocm` target:
+Then run the following command to compile with the `rocm` target backend:
```

Comment:
The flag says hip target backend. Should we try to stay consistent? Personally, I still don't understand the exact distinction between hip and rocm, and would assume it's even more confusing for newcomers.

What do you think of calling it a hip target and then linking to https://rocm.docs.amd.com/projects/HIP/en/docs-develop/what_is_hip.html?

Reply:
> What do you think of calling it a hip target

I think we should finish the discussions on https://github.com/iree-org/iree/pull/18477 and pick names we are happy with, then update the docs to reflect that.